sdk/src/types/api/core.rs

```diff
@@ -327,6 +327,8 @@ pub struct SubmitBlockResponse {
 pub enum BlockState {
     // Stored but not confirmed.
     Pending,
+    // TODO
+    Accepted,
     // Confirmed with the first level of knowledge.
```

Comment:
Also don't like the confirmed comment, `level of knowledge` says nothing without knowing exactly where this name comes from
Maybe something like the following would be better
```
accepted = Referenced by 67% of online validator committee.
confirmed = Referenced by 67% of entire validator committee.
```